src/bin/asr_recog_th.py

```diff
@@ -104,6 +109,10 @@ def cpu_loader(storage, location):
         return storage
     model.load_state_dict(torch.load(args.model, map_location=cpu_loader))
 
+    # read rnnlm
+    if args.rnnlm:
+        logging.warning("rnnlm integration is not implemented in the pytorch backend")
+
```

Comment:
Which is better, go through or exit?

Reply:
Go trough with warning is better not to change run.sh so much.